### PR TITLE
TransitionFade: allow for the region attribute to work properly

### PR DESCRIPTION
**Pull request by ninjamuffin99**
setting the 'region' didn't work. Quick little fixins so now it does.

### CODE REVIEW

**Comment by ninjamuffin99** [comment]
just for example


even when setting the region data attribute, the size of the fade would always be hardcoded to FlxG.width and FlxG.height
image

now it fills whatever region you set it as
image

(camera is zoomed out in these screenshots, so if it's hardcoded to FlxG.width, it will have gaps, shown in the first image)

**Comment by Gama11** [comment]
@larsiusprime Thoughts?

**Comment by larsiusprime** [comment]
Seems reasonable

File: flixel/addons/transition/TransitionFade.hx [hunk 1]
```diff
@@ -146,48 +146,48 @@ class TransitionFade extends TransitionEffect
 		}
 	}
 
-	function makeSprite(DirX:Float, DirY:Float):FlxSprite
+	function makeSprite(DirX:Float, DirY:Float, region:FlxRect):FlxSprite
```
Comment (Gama11) [inline flixel/addons/transition/TransitionFade.hx:149]: Um, CI seems to be failing because FlxRect isn't imported?

Comment (ninjamuffin99) [reply flixel/addons/transition/TransitionFade.hx:149]: woops! I just copy pasted my code from VS Code into Github web browser editor, forgot that!

**Comment by notdonxd** [comment]
So this is accepted and fixed?

File: flixel/addons/transition/TransitionFade.hx (hunk 1 above)
Comment (brosomethingwrongwithyothing) [reply flixel/addons/transition/TransitionFade.hx:150]: ok

**Pull request merged**